lib/compiler/semantic.js

```diff
@@ -847,7 +892,7 @@ class SemanticPass {
 
         if (module_ast !== undefined) {
             if (!this.modules.hasOwnProperty(modulename)) {
-                this.sa_ModuleDef(module_ast);
+                this.sa_Module(module_ast);
```

Comment:
This should be `sa_module` or better `sa_module_def` to not look like an AST node handler.